mlflow/protos/service.proto

```diff
@@ -4110,9 +4110,11 @@ message CreateGatewaySecret {
   optional string secret_value = 2;
```

Comment:
@BenWilson2 I think we should update the schema for secret_value here to support arbitrary key-value pair?